examples/text_labels_and_annotations/text_alignment.py

```diff
@@ -47,24 +49,24 @@
         verticalalignment='top',
         transform=ax.transAxes)
 
-ax.text(left, 0.5*(bottom + top), 'right center',
+ax.text(left, 0.5 * (bottom + top), 'right center',
```

Comment:
"If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies)."